stable/bitcoind/.helmignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Please use the standard .helmignore file.

Reply:
Sorry - updated.